scripts/perf-test/ec2-terminate.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+### BEGIN INIT INFO
```

Comment:
:question: I don't know what the next 8 lines of code do...? That aren't informative to me?